front/app/containers/ProjectsShowPage/timeline/VotingResults/ProgressBar.tsx

```diff
@@ -19,39 +19,51 @@ import { transparentize } from 'polished';
 // i18n
 import { useIntl } from 'utils/cl-intl';
 import useLocalize from 'hooks/useLocalize';
-import messages from 'components/VoteInputs/multiple/AssignMultipleVotesInput/messages';
+import messages from './messages';
+import assignMultipleVotesInputMessages from 'components/VoteInputs/multiple/AssignMultipleVotesInput/messages';
+
+import { roundPercentage } from 'utils/math';
+import { IIdeaData } from 'api/ideas/types';
 
 interface Props {
   phaseId: string;
-  votes?: number;
-  votesPercentage: number;
-  baskets?: number;
-  tooltip?: string;
+  idea: IIdeaData;
 }
 
-const ProgressBar = ({
-  phaseId,
-  votes,
-  votesPercentage,
-  baskets,
-  tooltip,
-}: Props) => {
+const ProgressBar = ({ phaseId, idea }: Props) => {
   const theme = useTheme();
   const { formatMessage } = useIntl();
   const localize = useLocalize();
   const { data: phase } = usePhase(phaseId);
 
   if (!phase) return null;
 
+  const votingMethod = phase.data.attributes.voting_method;
+  const baskets =
+    votingMethod === 'single_voting'
+      ? undefined
+      : idea.attributes.baskets_count ?? 0;
+  const ideaVotes = idea.attributes.votes_count ?? 0;
+  // for budgetting, this is total budget spent?
+  const totalVotes = phase?.data.attributes.votes_count;
+  const votesPercentage = totalVotes
+    ? roundPercentage(ideaVotes, totalVotes)
+    : 0;
+  const tooltip =
+    votingMethod === 'budgeting'
+      ? formatMessage(messages.budgetingTooltip)
+      : undefined;
+  const votes = votingMethod === 'budgeting' ? undefined : ideaVotes;
```

Comment:
This is going to be split in 2 components in a next PR, or at least make it clearer which is used for which voting method.